eureka_ml_insights/models/models.py

```diff
@@ -292,7 +292,8 @@ def create_request(self, text_prompt, query_images=None, *args, **kwargs):
         user_content = {"role": "user", "content": text_prompt}
         if query_images:
             if len(query_images) > 1:
-                raise ValueError("Llama vision model does not support more than 1 image.")
+                logging.warning("Llama vision model does not support more than 1 image.")
```

Comment:
Have you tested this? Return None from create_request, passes the None to get_response and I'm not sure how urllib.request.urlopen behaves if Request object is None.

As we discussed, please adjust this return value in a way that you can handle the situation properly in handle_request_error, and return do_return True so the request does not get attempted again.